axes/tests/test_handlers.py

```diff
@@ -203,8 +274,8 @@ def test_whitelist(self, log):
 
     def test_user_login_failed_multiple_username(self):
         configurations = (
-            (1, 2, {}, ["admin", "admin1"]),
-            (1, 2, {"AXES_USE_USER_AGENT": True}, ["admin", "admin1"]),
+            (2, 1, {}, ["admin", "admin1"]),
```

Comment:
Because `AccessAttempt` objects count changes with what I wrote in the description:

> I had to change the logic a bit. Now every attempt is logged/increased to separate database record based on IP, username and user_agent. This means, that the username is never rewritten with failed attempt as was previous behaviour.

It should still fail after same number of attempts, only the records in the database are more detailed in regards of what happened.